cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -331,6 +331,13 @@ func getAvailableResources(clientBuilder controller.ControllerClientBuilder) (ma
 			return false, nil
 		}
 
+		healthStatus := 0
+		client.Discovery().RESTClient().Get().AbsPath("/healthz").Do().StatusCode(&healthStatus)
+		if healthStatus != 200 {
+			glog.Errorf("Server isn't healthy yet.  Waiting a little while.")
```

Comment:
glog is used above.  This isn't unexpected and it is handled, so utilruntime.HandleError is inappropriate.